src/patterns/atoms/block/paragraph/paragraph.stories.js

```diff
@@ -0,0 +1,14 @@
+import paragraph from './paragraph.twig';
+
+export default {
+  title: 'Atoms|Block/Paragraph',
+};
+
+export const Basic = () => paragraph({
+  text: [
+    'It\'s not ',
+    { template: './atoms/inline/italic/italic.twig', arguments: { text: 'just' } },
```

Comment:
Should make this function, as it's not the easiest to read here.

Reply:
Done in https://github.com/libero/storybook/pull/14/commits/7fc2704e02e77e6c9b5bec78dfe3f611c708b89c

Reply:
Feels like this could be part of the webpack loader. So instead of exporting a single function to call:

```
import pattern from  './path/to/patter.twig';

pattern({ ... }); // render
```

You could have a render method, and potentially others:
```
import pattern from './path/to/pattern.twig';
import someOtherPattern from './path/to/pattern-2.twig';

pattern.render({ 
  other: someOtherPattern.reference({ ... })
});
```

Give me a shout if that sort of syntax would be of interest. Webpack loaders can be chained, so it could be an additive inclusion, not replacing the `twig-loader`.

Reply:
Interesting; could probably replace https://github.com/libero/storybook/blob/d7352f41bff7373d81f871aa17172ef2b61c94b5/.storybook/config.js#L10-L15 too.

Not entirely sure how I feel about this Webpack magic though. Any way to help IDEs understand it?

Reply:
Absolutely, Typescript definition files can be as descriptive as you like (the definition itself here is pseudo code):
```typescript
// ./@types/twig-import.d.ts
declare module "*.twig" {
  const render = (props: any) => string;
  const reference = <T extends {} = any>(props: any) => T;
  const twigFile: { render, reference };
  export default twigFile;
}
```

Reply:
You can go as far as declaring the APIs of individual twig templates

Reply:
Won't include in this PR, but food for thought 🤔